main.tf

```diff
@@ -137,7 +137,17 @@ resource "aws_identitystore_group_membership" "sso_group_membership" {
   identity_store_id = local.sso_instance_id
 
   group_id  = (contains(local.this_groups, each.value.group_name) ? aws_identitystore_group.sso_groups[each.value.group_name].group_id : data.aws_identitystore_group.existing_sso_groups[each.value.group_name].group_id)
-  member_id = (contains(local.this_users, each.value.user_name) ? aws_identitystore_user.sso_users[each.value.user_name].user_id : data.aws_identitystore_user.existing_sso_users[each.value.user_name].user_id)
+  member_id = aws_identitystore_user.sso_users[each.value.user_name].user_id
```

Comment:
May i ask you why this? User can be in aws_identitystore_user.sso_users in case of new user addition or data.aws_identitystore_user.existing_sso_users in case of already existing user... (maybe i'm missing something but wanted to check this)